### DISCUSSION TITLE
Code-server and Codeium's Chat Rendering Issue

### DISCUSSION THREAD

**Discussion by sinnos** [coder/code-server#6606] [Q&A]
Hello there,
i'm not sure if this is normal but my browser does not render the code-server's chat properly.
![image](https://github.com/coder/code-server/assets/155295577/a1435d3c-05b7-4113-b631-078fb5a8e84e)

however, when running visual studio code itself along with the codeium extension, it properly renders the chat like so:
![image](https://github.com/coder/code-server/assets/155295577/c939f347-5567-443d-9ef0-be8c12d64dc8)
What might I do to be able to render the chat within the browser? Or perhaps this is not yet supported in code-server yet?

**Comment by code-asher** [comment]
I installed Codeium on code-server v4.20.0 and it is working for me.  Maybe you need to update code-server?

**Reply by code-asher** [reply]
Weird though, it makes me log in every time I reload.

**Reply by sinnos** [reply]
I understand that Codeium works on code-server v4.20.0, and I am using the latest version:
![image](https://github.com/coder/code-server/assets/155295577/0f4d6cd3-8cb6-46ee-9aab-5be36deba5de)

However, I am unsure why the chat is not being loaded inside of the web browser. I am not sure how to work around this issue when using the browser. Is there a way to display the Codeium chat within a browser containing code-server?

**Reply by code-asher** [reply]
Sorry for being vague, what I meant is that chat is working for me.  It shows up, I can type questions, and it responds.  I was using code-server with Chromium.

Are there any logs in the browser console?

**Comment by sultanm900** [comment]
تاباؤتؤؤلت
